grails-app/services/org/bbop/apollo/SequenceService.groovy

```diff
@@ -39,22 +39,19 @@ class SequenceService {
      * @param feature
      * @return
      */
-    String getResiduesFromFeature(Feature feature ) {
-        List<FeatureLocation> featureLocationList = FeatureLocation.createCriteria().list {
-            eq("feature",feature)
-            order("fmin","asc")
+    String getResiduesFromFeature(Feature feature) {
+        String returnResidues = ""
+        def orderedFeatureLocations = feature.featureLocations.sort { it.fmin }
```

Comment:
Yep, I used this thing called "javaflamegraph" to identify those pain points https://www.npmjs.com/package/javaflamegraph

Pretty easy to setup, just found it today too...